src/DDTrace/Integrations/Curl/CurlIntegration.php

```diff
@@ -148,3 +182,16 @@ public static function injectDistributedTracingHeaders($ch)
         }
     }
 }
+
+/**
+ * @param span $span
+ * @param tagName $tagName
+ * @param info $info
+ */
+ function tagFromCurlInfo(&$span, &$info, $tagName, $curlInfoOpt)
```

Comment:
This should be a private method on the `CurlIntegration` class so we don't introduce a global symbol that could clash with user code. The span variable should not be passed by reference.

Reply:
Made this class scoped, but making it private caused the tests to fail with 

`Error: Call to private method DDTrace\Integrations\Curl\CurlIntegration::tagFromCurlInfo() from context 'DDTrace\Tests\Integrations\Curl\CurlIntegrationTest'`